### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by srilekha533**

### CODE REVIEW

File: src/settings/Settings.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      </div>
+  )
+}
+export default Settings;
+
+
+
+    //  let contacts = []
+        //  const searchkey = this.state.search.trim().toLowerCase();
+        //  if(searchkey && searchkey.length>0){
```
Comment (NagabrundaKodumuri) [inline src/settings/Settings.js:158]: remove commented code.

File: src/users/UserCreate.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+              console.log(posRes.data)
+         })
+             .catch(errRes=> {
+             console.log(errRes)
+             })
+            this.props.history.push('/user')
+        
+    }
+    render() {
+      const { username ,password, email, role, first_name,last_name,profile_pic,has_sales_access,has_sales_access_true,has_marketing_access_true,has_marketing_access,status }= this.state
```
Comment (NagabrundaKodumuri) [inline src/users/UserCreate.js:47]: make all this variables as a object.

**Comment by ashwin31** [comment]
@srilekha533 fix conflicts

File: src/users/User.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+       apiurl.get('/users/')
+    
+ .then((posRes) => {
+    this.setState({ 
+        usersList : posRes.data.active_users,
+        inactiveuser : posRes.data.inactive_users
+        
+        
+       })
+   // console.log(posRes.data);
```
Comment (NagabrundaKodumuri) [inline src/users/User.js:29]: remove console.log

File: src/users/User.js [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+      actions = () => {
+          this.setState({
+              show : !this.state.show
+          })
+      }
+
+    render(){
+         const { usersList,inactiveuser , term } = this.state
+        // console.log(usersList.length)
```
Comment (NagabrundaKodumuri) [inline src/users/User.js:55]: remove console.log's all over the app .

File: src/users/User.js [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+                        <td>{user.email}</td>
+                        <td>{user.role}</td>
+                        <td>
+                            { user.has_marketing_access || user.has_sales_access ? 'sales,marketing' : 'null' }
+                           
+                        </td>
+                        <td> 
+                        <Link to={`/users/status/${user.id}`} id={index}  >Active</Link>
+
+                            {/* // {user.is_active? 'Active' : 'inactive' } */}
```
Comment (NagabrundaKodumuri) [inline src/users/User.js:124]: Remove console.log

File: src/users/UserDelete.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+  componentDidMount(){
+    const id = this.props.match.params.id
+            apiurl.get(`/users/${id}/`)
+            .then((posRes) => {
+              this.setState({  
+                users :  posRes.data.data.user_obj
+               
+              })
+               console.log(posRes.data.data.user_obj)
```
Comment (NagabrundaKodumuri) [inline src/users/UserDelete.js:20]: Remove console.log

File: src/users/UserEdit.js [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+    username : '',
+    first_name : '',
+    last_name : '',
+    profile_pic: null,
+    has_sales_access : false,
+    has_sales_access_null: true,
+    has_marketing_access : false,
+    has_marketing_access_null : true,
+    is_active : true
+  //  user : ''
```
Comment (NagabrundaKodumuri) [inline src/users/UserEdit.js:19]: remove commented code

**Review by NagabrundaKodumuri — COMMENTED** [review]
Please remove all console.log's and commented code before pushing the code to git.

File: src/profile/ProfileDetails.js [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    componentDidMount(){
+    
+        apiurl.get('/profile/')
+        .then(pos => {
+            this.setState({
+                profile: pos.data.user_obj
+            })
+            console.log(pos.data.user_obj)
```
Comment (NagabrundaKodumuri) [inline src/profile/ProfileDetails.js:18]: please remove console.log

File: src/profile/ProfileDetails.js [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+        }).catch(err=>{
+            console.log(err)
```
Comment (NagabrundaKodumuri) [inline src/profile/ProfileDetails.js:21]: please remove console.log

File: src/users/User.js [hunk 10]
```diff
@@ -0,0 +1,220 @@
...
+ .then((posRes) => {
+    this.setState({ 
+        usersList : posRes.data.active_users,
+        inactiveuser : posRes.data.inactive_users
+        
+        
+       })
+   
+ }, (errRes)=> {
+     console.log(errRes)
```
Comment (NagabrundaKodumuri) [inline src/users/User.js:31]: please remove console.log

File: src/users/UserEdit.js [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+                last_name : posRes.data.data.user_obj.last_name,
+                username : posRes.data.data.user_obj.username,
+                profile_pic : posRes.data.data.user_obj.profile_pic,
+                has_marketing_access : posRes.data.data.user_obj.has_marketing_access,
+                has_sales_access : posRes.data.data.user_obj.has_sales_access,
+                is_active: posRes.data.data.user_obj.is_active
+              })
+               
+              }).catch(errRes=> {
+                         console.log(errRes)
```
Comment (NagabrundaKodumuri) [inline src/users/UserEdit.js:34]: please remove console.log

File: src/users/inactiveUsers/InactiveUser.js [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+    }
+
+componentDidMount(){
+       apiurl.get('/users/')
+    
+ .then((posRes) => {
+    this.setState({ 
+        inactiveuser : posRes.data.inactive_users
+       })
+     // console.log(this.state.inactiveuser)
```
Comment (NagabrundaKodumuri) [inline src/users/inactiveUsers/InactiveUser.js:24]: please remove commented code

File: src/users/inactiveUsers/InactiveUser.js [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+ }, (errRes)=> {
+     console.log(errRes)
```
Comment (NagabrundaKodumuri) [inline src/users/inactiveUsers/InactiveUser.js:26]: please remove console.log

**Review by NagabrundaKodumuri — APPROVED** [review]

File: src/settings/Modal.js [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+//           <span aria-hidden="true">&times;</span>
+//         </button>
+//       </div>
+//       <div class="modal-body">
+//         <p>Are You Sure You Want To Delete This?</p>
+//       </div>
+//       <div class="modal-footer">
+//         <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+//         <button type="button" class="btn btn-primary">Delete</button>
+//       </div>
```
Comment (NagabrundaKodumuri) [inline src/settings/Modal.js:22]: delete this file if it is not required.

File: src/settings/blockedEmail/EditBlockedEmail.js [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+            apiurl.get(`/settings/block-emails/`)
+            .then((posRes) => {
+              this.setState({ 
+               emails: posRes.data.blocked_emails
+            
+              })
+              
+               
+              }).catch(errRes=> {
+                       console.log(errRes)
```
Comment (NagabrundaKodumuri) [inline src/settings/blockedEmail/EditBlockedEmail.js:24]: remove console.log

File: src/settings/blockedomain/EditBlockedDomain.js [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+ 
+            apiurl.get(`/settings/block-domains/`)
+            .then((posRes) => {
+              this.setState({ 
+               emails: posRes.data.blocked_domains
+            
+              })
+               
+              }).catch(errRes=> {
+                       console.log(errRes)
```
Comment (NagabrundaKodumuri) [inline src/settings/blockedomain/EditBlockedDomain.js:23]: remove all console.log's in this pr

File: src/settings/contacts/ContactFilter.js [hunk 17]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ShowMe = (props) => {
+
+
+    return(
+        <div className="container">
+          {/* <div className="row">
```
Comment (NagabrundaKodumuri) [inline src/settings/contacts/ContactFilter.js:9]: remove commented code if not requried

File: src/settings/contacts/DeleteContact.js [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+class DeleteContact extends React.Component{
+
+  state= {
+    contacts : []
+  }
+
+//   componentDidMount(){
+    // // const id = this.props.match.params.id
+    //         apiurl.get(`/settings/contacts/`)
+    //         .then((posRes) => {
```
Comment (NagabrundaKodumuri) [inline src/settings/contacts/DeleteContact.js:15]: remove commented code if not required

File: src/users/User.js [hunk 19]
```diff
@@ -0,0 +1,239 @@
...
+    
+ .then((posRes) => {
+    this.setState({ 
+        usersList : posRes.data.active_users,
+        inactiveuser : posRes.data.inactive_users
+        
+        
+       })
+     
+      console.log(posRes.data)
```
Comment (NagabrundaKodumuri) [inline src/users/User.js:31]: remove console.log

**Review by NagabrundaKodumuri — APPROVED** [review]

**Comment by ashwin31** [comment]
@srilekha533 fix conflicts

**Pull request merged**